contributing.md

```diff
@@ -0,0 +1,67 @@
+Contributing
+============
+
+## Issue Tracking
+
+New feature requests, changes, enhancements, non-methodology features, and bug reports can be filed as new issues in the
+[GitHub.com issue tracker](https://github.com/entralliance/entr_runtime/issues) at any time. Please be sure to fully describe the
+issue.
```

Comment:
I'm not sure how to set this up, but I've seen issue templates like [this](https://github.com/dbt-labs/dbt-core/issues/new?assignees=&labels=enhancement%2Ctriage&template=feature-request.yml&title=%5BFeature%5D+%3Ctitle%3E) for opening a new issue to make sure people provide the right info to get conversation/development started

Reply:
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms